src/azul/indexer/document_service.py

```diff
@@ -68,6 +72,8 @@ def field_type(self, catalog: CatalogName, path: Tuple[str, ...]) -> FieldType:
                 raise KeyError(f'Path {path} not represented in field_types')
             except TypeError:
                 raise TypeError(f'Path {path} not represented in field_types')
+            if isinstance(field_types, list):
+                field_types = one(field_types)
             if field_types is None:
                 return None
```

Comment:
PyCharm emits a type warning here. Should the return type annotation be adjusted?

Reply:
I ran tests and check the coverage report and it looks like this line is never touched https://codecov.io/gh/DataBiosphere/azul/pull/2697/diff?src=pr&el=tree#D3-74.

I traced it back to this PR https://github.com/DataBiosphere/azul/pull/1382/files#diff-73be64e9409fcaf8430fde4dd1a7e10f31d269606de6438161de8034281228d5R76.

I'll ask @danielsotirhos about this, but otherwise I think it should go.

Reply:
From the Slack conversation with @danielsotirhos, I'm concluding that this is dead code and have removed it https://ucsc-gi.slack.com/archives/C705Y6G9Z/p1610739151024300.